docker-compose.yml

```diff
@@ -19,18 +21,26 @@ services:
       context: ./
       dockerfile: Dockerfile
       args:
-        - VIVO_DIR=/usr/local/vivo/home
-        - TDB_FILE_MODE=direct
-        - SOLR_URL=http://solr:8983/solr/vivocore
+        - TDB_FILE_MODE=${VIVO_TDB_FILE_MODE}
+        - VIVO_HOME=${VIVO_CONTAINER_VIVO_HOME}
+        - SOLR_URL=http://solr:${SOLR_CONTAINER_PORT}/solr/vivocore
     environment:
-      - RESET_HOME=${RESET_HOME}
-      - VERBOSE=${VERBOSE}
+      - RESET_HOME=${VIVO_RESET_HOME}
+      - VERBOSE=${VIVO_VERBOSE}
     ports:
-      - 8080:8080
+      - ${VIVO_HOST_PORT}:${VIVO_CONTAINER_PORT}
     volumes:
-      - ${LOCAL_VIVO_HOME}:/usr/local/vivo/home
+      - ${VIVO_HOST_VIVO_HOME}:${VIVO_CONTAINER_VIVO_HOME}
     networks:
       - vivo
+    depends_on:
+      - solr
+    entrypoint:
```

Comment:
Is it a better practice to add a healthcheck as described here (https://medium.com/@nikjos/docker-compose-health-check-for-apache-solr-251c04775e24) to Solr container, and then just say that the depends_on condition for tomcat container is "service_healthy" Solr container? IMO, that would remove the need for custom check script and improve potential integration with orchestration tools if somebody decides to use one.

Reply:
Yes absolutely. Shortly after I put in this PR I did exactly that with another project with docker compose.

Would this be a reasonable health check for Solr?
```
    healthcheck:
      test: ["CMD-SHELL", "curl -Isf http://localhost:8983/solr/vivocore/admin/ping || exit 1"]
      interval: 15s
      timeout: 5s
      retries: 3
...
    depends_on:
      solr:
        condition: service_healthy
```

Reply:
FYI, I read that same article but had difficulty with the health check command they used. I will test a bit before adding additional commit.

Reply:
> Yes absolutely. Shortly after I put in this PR I did exactly that for another project with docker compose.
> 
> Would this be a reasonable health check for Solr?
> 
> ```
>     healthcheck:
>       test: ["CMD-SHELL", "curl -Isf http://localhost:8983/solr/vivocore/admin/ping || exit 1"]
>       interval: 15s
>       timeout: 5s
>       retries: 3
> ...
>     depends_on:
>       solr:
>         condition: service_healthy
> ```

This seems like a good approach, nice work.